### PR TITLE
Pavanh007 patch 1

### PR DESCRIPTION
**Pull request by pavanh007**
## Fixes Issue
Fixed the changes required

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7201"></a>

### CODE REVIEW

File: pavanh007-initial-commit [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "Backend developer | Tech enthusiast",
+    "avatar": "https://i.ibb.co/vHWZ959/linkedIn.png"
```
Comment (ChinmayMhatre) [inline pavanh007-initial-commit:6]:
```suggestion
```
Avatar property is no longer needed

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please change the filename to `pavanh007.json` and move it to the `data` folder

File: pavanh007-initial-commit [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline pavanh007-initial-commit:4]: In addition to the above requested changes, please remove `type`, & `displayStatsPublic` keys from your `json` as we no longer need them. 

[Edit Your File Here](https://github.com/pavanh007/LinkFree/edit/pavanh007-patch-1/pavanh007-initial-commit)

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by SaraJaoude** [comment]
Temporary branch was not merged as exact Profile already exists: https://github.com/EddieHubCommunity/LinkFree/blob/main/data/pavanh007.json

**Pull request merged**